A/Arb/build_tarballs.jl

```diff
@@ -22,24 +22,21 @@ import Pkg.Types: VersionSpec
 # to all components.
 
 name = "Arb"
-upstream_version = v"2.21.1"
-build_for_julia16_or_newer = true
-version_offset = build_for_julia16_or_newer ? v"0.0.1" : v"0.0.0"
+upstream_version = v"2.22.0"
+version_offset = v"0.0.1"
```

Comment:
```suggestion
version_offset = v"0.0.0"
```
I think we can reset this to `0.0.0`, right @fingolfin?